examples/stereo_reconstruction_core_and_direction.py

```diff
@@ -0,0 +1,494 @@
+            alpha=0.7,
+            title="SHOWER'S CORE POSITION (GROUNDFRAME)",
+            radius=None,
+        )
+        array_disp.add_labels()
+        array_disp.set_line_hillas(parametrized_images, core_dict, range=100)
+
+        plt.plot(
+            simulated_core.x,
+            simulated_core.y,
+            "*",
+            color="black",
+            label="simulated core",
+        )
+        plt.plot(
+            reconstructed_core.x,
+            reconstructed_core.y,
+            "o",
+            color="red",
+            alpha=0.5,
+            fillstyle="none",
+            markersize=5,
+            label="reconstructed core",
+        )
+
+        plt.legend(loc="best")
+
+        # COORDINATES OF THE SHOWER'S DIRECTION IN THE SKY
+
+        # Transform all positions into the nominal frame
+        nominal_frame = NominalFrame(origin=array_pointing)
+        simulated_direction = SkyCoord(
+            alt=event.simulation.shower.alt,
+            az=event.simulation.shower.az,
+            frame="altaz",
+        ).transform_to(nominal_frame)
+        reconstructed_direction = SkyCoord(
+            alt=event.dl2.shower["HillasReconstructor"].alt,
+            az=event.dl2.shower["HillasReconstructor"].az,
+            frame="altaz",
+        ).transform_to(
+            nominal_frame
+        )  # coming from TelescopeFrame, axes are flipped!
+
+        error_direction = simulated_direction.separation(reconstructed_direction)
+
+        print("COORDINATES OF THE SHOWER'S DIRECTION IN THE SKY:")
+        print(
+            f"- simulated : \
+        lon = {simulated_direction.fov_lon}, \
+        lat = {simulated_direction.fov_lat}"
+        )
+        print(
+            f"- reconstructed : \
+        lon = {reconstructed_direction.fov_lon}, \
+        lat = {reconstructed_direction.fov_lat}"
+        )
+        print(f"- error = {error_direction}")
+
+        # plot of the shower's direction in the nominal frame
+
+        plt.subplot(n_rows, n_cols, 2, aspect=1)
+        plt.title("SHOWER'S DIRECTION IN THE SKY (NOMINAL FRAME)")
+        plt.xlabel("Field of View Longitude [deg]")
+        plt.ylabel("Field of View Latitude [deg]")
+        ax = plt.gca()
+        cmap = plt.cm.get_cmap("tab20c", len(parametrized_images))
+
+        for i, tel_id in enumerate(parametrized_images.keys()):
+
+            # image = np.zeros_like(parametrized_images[tel_id])
+
+            telescopeframe = TelescopeFrame(
+                telescope_pointing=telescope_pointings[tel_id]
+            )
+
+            # Plot camera
+            nominal_geometry = geometry_TelescopeFrame[tel_id].transform_to(
+                nominal_frame
+            )
+            pixel_coord = SkyCoord(
+                fov_lon=nominal_geometry.pix_x,
+                fov_lat=nominal_geometry.pix_y,
+                frame=nominal_frame,
+            )
+
+            # Plot Hillas ellipses
+
+            centerOfgravity_telescopeFrame = SkyCoord(
```

Comment:
That is a bit inconsistent in terms of variable naming

Reply:
done